src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -107,6 +107,7 @@ protected Object[][] getContents() {
         {"R_statementMustBeExecuted", "The statement must be executed before any results can be obtained."},
         {"R_modeSuppliedNotValid", "The supplied mode is not valid."},
         {"R_errorConnectionString", "The connection string contains a badly formed name or value."},
+        {"R_errorServerName", "The server name {0} is badly formed."},
```

Comment:
in looking at the other error string relating to connection properties, this should be "The serverName connection property {0} is badly formed."